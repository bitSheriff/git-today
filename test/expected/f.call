../target/debug/git-today repos/f
