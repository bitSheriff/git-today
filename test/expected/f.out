╭─────────┬──────────────╮
│ Author  ┆ # of Commits │
╞═════════╪══════════════╡
│ Author1 ┆      10      │
╰─────────┴──────────────╯
╭─────────────┬──────────────╮
│ Issue Type  ┆ # of Commits │
╞═════════════╪══════════════╡
│ 🐛 Bugs     ┆       1      │
├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ 🚀 Features ┆       7      │
├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ 📝 Docs     ┆       2      │
╰─────────────┴──────────────╯
